app/models/paper.rb

```diff
@@ -10,7 +10,7 @@ class Paper < ApplicationRecord
   has_many :links, dependent: :destroy
   has_many :api_import_responses, dependent: :destroy
 
-  accepts_nested_attributes_for :authors, reject_if: proc { |attributes| attributes['name'].blank? }
+  accepts_nested_attributes_for :authors, reject_if: proc { |attributes| attributes['last_name'].blank? }
```

Comment:
In the long run, dunno if we want to consider how to deal with foreign names.  Perhaps last name should be family name and first name should be given name.